src/components/grid/grid-container/grid-container.style.js

```diff
@@ -0,0 +1,30 @@
+import styled from 'styled-components';
+
+const GridContainerStyle = styled.div`
+  display: grid;
+  grid-template-columns: repeat(12, 1fr);
+  grid-template-rows: auto;
+  width: auto;
+  margin: 40px;
+  grid-gap: 40px;
+  
+  @media screen and (max-width: 1920px) {
+    grid-gap: 24px;
+  }
+  
+  @media screen and (max-width: 1259px) {
+    grid-gap: 24px;
```

Comment:
this is probably redundant as it inherits grid-gap: 24px; from 1920px media query.